get_started.csolution.yml

```diff
@@ -6,9 +6,9 @@ solution:
   cdefault:
 
   select-compiler:
-    - compiler: GCC
+#    - compiler: GCC
```

Comment:
What changes if you comment out the compilers from select-compiler?
I would leave them in. This is a clear indicator that these compilers are supported as well.